src/config.js

```diff
@@ -3,7 +3,7 @@
 // './queries' are query functions.
 let config = {
   testIdAttribute: 'data-testid',
-  asyncUtilTimeout: 4500,
+  asyncUtilTimeout: 1000,
```

Comment:
Just realized I amended the first of those two changes. So this is all that first change was.